models/optional_modules/consent/snowplow_web_consent_scope_status.sql

```diff
@@ -23,7 +23,7 @@ with arrays as (
   )
 
 select
-  replace(consent_scope,'"', '') as scope,
+  replace(replace(replace(consent_scope, '"', ''), '[', ''), ']', '') as scope,
   count(*) as total_consent
 
```

Comment:
What's the reaosn for this change? I assume something is returning a list (of one thing?) when it wasn't before? Feels like we're maybe doing something wrong above if this is only a single item list?

Reply:
The {{ snowplow_utils.get_split_to_array }} macro takes the string and creates an array out of it so that we can unnest it later. Perhaps the unnest macro should take care of this cleanup.. I checked snowflake and there everything became double quoted after the unnest macro. Perhaps we did not catch this before as the media player unnests integers, not sure.

Reply:
So the update on this was that the unnest macro was originally done with a select statement but not wrapped inside a cte. In order to do any replacements or format changes we would need to wrap this in a separate cte and another one which takes care of that. Due to backward compatibility the decision for now is to handle this afterwards in the models themselves. Another impact we have not anticipated beforehand is that Redshift creates a super data type as a result of this unnesting. If we need string replacement working with that field it is not going to be possible, it needs to be casted back to string afterwards to do that.